From a0028a470973cf7247c4aecf9d43ceca0382a637 Mon Sep 17 00:00:00 2001
From: "Alam, SahibeX" <sahibex.alam@intel.com>
Date: Tue, 2 Jan 2024 12:15:57 +0530
Subject: [PATCH] CVE_2022_20234 STS test fixed. 

Failed due to Notification Access Confirmation Activity
should not be started from outside of system process. 

NotificationAccessConfirmationActivity is expected to false.

Test done: STS r22 Test Passed
TC:android.security.cts.CVE_2022_20234#testPocCVE_2022_20234

Tracked-On: OAM-114647
Signed-off-by: Alam, SahibeX <sahibex.alam@intel.com>
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 2af2db488..d1088eb23 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -911,7 +911,7 @@
         </activity>
 
         <activity android:name=".notifications.NotificationAccessConfirmationActivity"
-            android:exported="true"
+            android:exported="false"
             android:taskAffinity="com.android.car.settings.notifications.NotificationAccessConfirmationActivity"
             android:theme="@style/ActionDialogTheme"
             />
-- 
2.17.1

